README.md

```diff
@@ -2,54 +2,49 @@
 
 # Project Name
```

Comment:
@TedLivist  it already changed in the awesome branch!

Reply:
@yosefHesham I don't think so. This is your `awesome` branch.

  ![yt-1](https://user-images.githubusercontent.com/36542986/165131641-8adb814f-d5d9-4e7c-9837-9308dbfe18d8.jpg)

Reply:
![Screenshot_2022-04-25-19-05-46-412_com android chrome](https://user-images.githubusercontent.com/45171920/165138472-b6b6bfca-41ff-40c7-9b86-18e3799e70d7.jpg)
So you want a name more descriptive than that?
@TedLivist

Reply:
@yosefHesham yeah. Just change that Project Name to Awesome Book as you have below it

Reply:
You got a point of view actually.
But is this necessary? Suppose I want to have like a heading meta data (project name) :..... 
It's kind of preferences I guess and it's not a core requirement
Am I wrong? 
@TedLivist

Reply:
@yosefHesham it is a core requirement to have descriptive headers in your documentation. However, since you implemented that using this style I guess that's okay.

I just think it's redundant though.

Reply:
It's already descriptive header , and yeah it might be redundant, my partner actually did that not me, but I think it would be better if you suggested it as an optional, not preventing the merge .
Thanks very much for you effort any way.
@TedLivist